backend/sprintone/src/main/java/com/project/omega/bean/Industry.java

```diff
@@ -0,0 +1,86 @@
+package com.project.omega.bean;
+
+import lombok.Builder;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+@Entity
+@Builder
+public class Industry {
```

Comment:
Can we call this `Supplier` instead, I think it might make more sense?

Reply:
Agree with the naming convention